docs/agents/sematext-agent/kubernetes/hostnetwork.md

```diff
@@ -0,0 +1,22 @@
+title: `hostNetwork` Setting Usage for Control Plane Components Monitoring
+description: Explains why we need `hostNetwork` access and how to turn it off if desired.
+
+## What is `hostNetwork`?
+
+In Kubernetes, the `hostNetwork` is a configuration option that allows a container to share the network namespace of its host node. When `hostNetwork` is enabled for a pod, it means that the pod's containers will use the networking stack of the underlying host machine instead of having their own isolated network stack.
+
+## Sematext Agent and `hostNetwork` Access
+
+Sematext Agent requires `hostNetwork` access to monitor the control plane components. By default, the `hostNetwork` setting is set to `true` starting from version 3.0.0 of Sematext Agent.
```

Comment:
@boratanrikulu `hostnetwork` access or enabled?

Reply:
Hmm, I think they're both OK. Do you think `enabled` would be better?

Reply:
I updated it like this

```
## Sematext Agent and `hostNetwork` Access

Sematext Agent requires `hostNetwork: true` setting to monitor the control plane components. By default, the `hostNetwork` setting is set to `true` starting from version 3.0.0 of Sematext Agent.
```

Reply:
Looks good now.